src/plugins/drawing/components/drawing-layer.tsx

```diff
@@ -368,10 +345,9 @@ export class DrawingLayerView extends React.Component<DrawingLayerViewProps, Dra
           onDrop={this.handleDrop} >
 
         <svg xmlnsXlink="http://www.w3.org/1999/xlink" width={1500} height={1500} ref={this.setSvgRef}>
-          {this.renderObjects(object => object.type === "image" && !idsBeingDragged.includes(object.id))}
-          {this.renderObjects(object => object.type !== "image" && !idsBeingDragged.includes(object.id))}
-          {this.state.objectsBeingDragged.map((object) => renderDrawingObject(object))}
-          {this.renderSelectedObjects(objectsToRenderSelected, SELECTION_COLOR)}
+          {this.renderObjects(object => object.type === "image" )}
+          {this.renderObjects(object => object.type !== "image" )}
+          {this.renderSelectedObjects(this.state.selectedObjects, SELECTION_COLOR)}
```

Comment:
Something this will probably change is that the dragged objects will no longer be "on top" off everything else.  This might be fine to leave until we have z-order feature. With the z-order feature we'll need a better way to deal with the ordering when rendering the objects.

Or you could add a view to the object indicating it is being dragged by checking if dragX or dragY is set. And then use that to put the dragged objects on the top.

Reply:
So in the master branch, the object is drawn on top _while you are dragging it_, but then reverts to its old z-index when you drop it in the new position.  I think maintaining the actual z-index during drag is an improvement in terms of having behavior match expectations.

I could see the argument that we should change it so that moving an object always leaves it on top - like moving papers on a desk you usually drop them on top of existing piles, or some such metaphor.  But it also feels like a bit of an unexpected side-effect that could cause frustration.  Say, you just wanted to slide the background object over a bit and then suddenly it's on top and you have no way to push it back down.

Reply:
@bgoldowsky I think you might be right, so it seems fine to leave how you've got it in the PR. 
Can you make a PT story for this refactoring, and put a note in there about how the z-order while dragging behavior is now different. This way Tejal and Leslie will know this behavior has been changed as part of whatever CLUE release this goes out with.

Reply:
Created #185649749